bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Search.java

```diff
@@ -79,6 +80,17 @@ default String getRelativePath() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Gets the path of the search root.
```

Comment:
I would add that its the localized path: `Gets the path of the localized search root.`